### ISSUE TITLE
About detection network training

### ISSUE THREAD

**Issue by Ryeong26** [DrCoffey/DeepSqueak#107]
Hello

I am trying to analyze courtship calls of mice, so I first attempted to use mouse neural network and I figured out some missed calls. So I am trying now to train this neural network with my USV recording data. However, after choosing pre-trained network that I want to use, the program shows no response at all. 

I noticed that this process only work in 2018a in 2019(but I use 2021a ver), so I just wondered that this kind of bug(?) is not updated yet.

Thank you for your help.

**Comment by DrCoffey** [comment]
Hey Ryeong26,

Version 3 is now in development and it is far superior to v2.6

Download it here:
https://github.com/DrCoffey/DeepSqueak/tree/Screener

You may not need to train a new network in V3, but if you do the network architecture and image normalization is vastly improved.

**Comment by DrCoffey** [comment]
Version 3 is now available on the main branch. Let me know if you have any issues detecting calls or training networks with V3. Be sure to follow the Wiki when training a new network. I won't be supporting V2.6.2 anymore, so open a new issue if you have V3 issues,.
